e2e_playwright/conftest.py

```diff
@@ -248,6 +249,22 @@ def app(page: Page, app_port: int) -> Page:
     return page
 
 
+@pytest.fixture(scope="function")
+def app_with_query_params(
+    page: Page, app_port: int, request: FixtureRequest
+) -> Tuple[Page, Dict]:
+    """Fixture that opens the app with additional query parameters.
+    The query parameters are passed as a dictionary in the 'query_params' key of the request.
```

Comment:
This doesn't seem correct -- the line below gets the query params from `request.param`, not `request.query_params`